### DISCUSSION TITLE
Data format for submission

### DISCUSSION THREAD

**Discussion by kkchandr** [ARM-Development/comble-mip#45] [General]
Is submitting the model output data in DEPHY format mandatory for the COMBLE intercomparison? I'm unfamiliar with the DEPHY format. All CM1 outputs are in NetCDF format. It would be easier to submit the data in NetCDF.

**Comment by twjuliano** [comment]
Hello @kkchandr,
Thanks for reaching out with this question. In short, the output file is a netCDF. The DEPHY formatting corresponds to the variable naming conventions, etc. For the output variables, we actually deviated from DEPHY in some instances to suit our needs. If you take a look at the 'Output Conversion Notebooks' tab on the project website, you'll see example scripts to convert from raw model output to the DEPHY output file. We suggest building off one of those to help you get started as we've done a lot of the legwork.
Hope this info helps,
Tim

**Reply by kkchandr** [reply]
Thanks Tim! It makes sense.
